client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -22,6 +22,7 @@ type Props = ProductStoreBaseProps & {
 	isVisible: boolean;
 	duration: Duration;
 	onClose: () => void;
+	onProductChange: ( product: SelectorProduct | null ) => void;
```

Comment:
```suggestion
	onChangeProduct: ( product: SelectorProduct | null ) => void;
```

Reply:
Thanks for spotting this.. this is resolved. 👍